test/unit/lib/DiscoveryStreamFeed.test.js

```diff
@@ -145,7 +145,7 @@ describe("DiscoveryStreamFeed", () => {
 
       await feed.fetchFromEndpoint("https://getpocket.cdn.mozilla.net/dummy?consumer_key=$apiKey");
 
-      assert.calledWith(fetchStub, "https://getpocket.cdn.mozilla.net/dummy?consumer_key=replaced", {credentials: "omit"});
+      assert.calledWith(fetchStub, "https://getpocket.cdn.mozilla.net/dummy?consumer_key=replaced");
```

Comment:
If you don't want an exact comparison, you should be able to use `calledWithMatch` to still check for an object that at least has the "credentials" key with "omit" value

Reply:
Thanks!